pkg/config/lima_config_applier.go

```diff
@@ -23,25 +23,31 @@ const (
 	sociInstallationProvisioningScriptHeader = "# soci installation and configuring"
 	sociFileNameFormat                       = "soci-snapshotter-%s-linux-%s.tar.gz"
 	sociDownloadURLFormat                    = "https://github.com/awslabs/soci-snapshotter/releases/download/v%s/%s"
+	sociServiceDownloadURLFormat             = "https://raw.githubusercontent.com/awslabs/soci-snapshotter/v%s/soci-snapshotter.service"
 	sociInstallationScriptFormat             = `%s
 if [ ! -f /usr/local/bin/soci ]; then
 	# download soci
 	set -e
 	curl --retry 2 --retry-max-time 120 -OL "%s"
 	# move to usr/local/bin
 	tar -C /usr/local/bin -xvf %s soci soci-snapshotter-grpc
-fi
+	# changing containerd config
+	export config=etc/containerd/config.toml
+	echo "    [proxy_plugins.soci]
+	type = \"snapshot\"
+	address = \"/run/soci-snapshotter-grpc/soci-snapshotter-grpc.sock\" " >> $config
 
-# changing containerd config
-export config=etc/containerd/config.toml
-echo "    [proxy_plugins.soci]
-  type = \"snapshot\"
-  address = \"/run/soci-snapshotter-grpc/soci-snapshotter-grpc.sock\" " >> $config
+	# install as a systemd service
+	curl --retry 2 --retry-max-time 120 -OL "%s"
+	mv soci-snapshotter.service /usr/local/lib/systemd/system/
+	ln -s /usr/local/lib/systemd/system/soci-snapshotter.service /etc/systemd/system/multi-user.target.wants/
+	restorecon -v /usr/local/lib/systemd/system/soci-snapshotter.service
+	systemctl daemon-reload
```

Comment:
nit: should we also add `sudo systemctl add-requires soci-snapshotter.service containerd.service` here?

Reply:
sure, added in latest rev